src/math.rs

```diff
@@ -2300,13 +2364,102 @@ mod tests {
                 elevation: d(-10.)
             })
             .unwrap()
-                * enu_to_frd_pointing_north_east_and_up,
+                * enu_to_frd_pointing_north_east_and_up_rolled_upright,
             Bearing::<Frd>::build(Components {
-                azimuth: d(-1.2855),
-                elevation: d(39.9944)
+                azimuth: d(1.2855),
+                elevation: d(-39.9944)
             })
             .unwrap(),
             epsilon = 0.0001_f64.to_radians() // FRD bearing is approximate
         );
+
+        // assume forward is pointing North
+        let enu_to_frd_pointing_north_inverted =
+            unsafe { Rotation::<Enu, Frd>::from_tait_bryan_angles(d(90.), d(0.), d(0.)) };
+
+        // a bearing pointing 3° off East
+        assert_relative_eq!(
+            Bearing::<Enu>::build(Components {
+                azimuth: d(93.),
+                elevation: d(0.)
+            })
+            .unwrap()
+                * enu_to_frd_pointing_north_inverted,
+            Bearing::<Frd>::build(Components {
+                azimuth: d(267.),
+                elevation: d(0.)
+            })
+            .unwrap()
+        );
+
+        // assume forward is pointing North East at a 30° elevation but inverted
+        let enu_to_frd_pointing_north_east_and_inverted = unsafe {
+            Rotation::<Enu, Frd>::from_tait_bryan_angles(
+                d(45.),
+                d(-30.), // rotating FRD to point up
+                d(0.),
+            )
+        };
+
+        // a bearing pointing North East and Down
+        assert_relative_eq!(
+            Bearing::<Enu>::build(Components {
+                azimuth: d(45.),
+                elevation: d(-10.)
+            })
+            .unwrap()
+                * enu_to_frd_pointing_north_east_and_inverted,
+            Bearing::<Frd>::build(Components {
+                azimuth: d(0.),
+                elevation: d(40.)
+            })
+            .unwrap()
+        );
+    }
+
+    #[test]
+    fn enu_to_ned_equivalent_gives_same_rotation() {
+        let lat = d(52.);
+        let lon = d(-3.);
+
+        // Check quaternion equality of chained rotation
+        let ecef_to_ned = unsafe { Rotation::<Ecef, PlaneNed>::ecef_to_ned_at(lat, lon) };
+        let ecef_to_enu = unsafe { Rotation::<Ecef, PlaneEnu>::ecef_to_enu_at(lat, lon) };
+        let ned_from_enu = unsafe { ecef_to_enu.into_ned_equivalent::<PlaneNed>() };
```

Comment:
Maybe call this `ecef_to_ned_via_enu`? And ditto for the equivalent in the other test?